source/classes.tex

```diff
@@ -1445,17 +1445,20 @@
 \end{note}
 
 \pnum
-\indextext{constructor!implicitly called}%
-Default constructors are called implicitly to create class objects of static, thread,
-or automatic storage duration~(\ref{basic.stc.static}, \ref{basic.stc.thread}, \ref{basic.stc.auto}) defined
-without an initializer\iref{dcl.init},
-are called to create class objects of dynamic storage duration\iref{basic.stc.dynamic} created by a
-\grammarterm{new-expression}
-in which the
-\grammarterm{new-initializer}
-is omitted\iref{expr.new}, or
-are called when the explicit type conversion syntax\iref{expr.type.conv} is
-used.
+\indextext{constructor!implicitly invoked}%
+A default constructor is invoked implicitly
+\begin{itemize}
+\item
+to initialize a class object of static, thread,
+or automatic storage duration~(\ref{basic.stc.static}, \ref{basic.stc.thread}, \ref{basic.stc.auto})
+defined without an initializer\iref{dcl.init},
+\item
+to initialize a class object of dynamic storage duration\iref{basic.stc.dynamic}
+created by a \grammarterm{new-expression}
+in which the \grammarterm{new-initializer} is omitted\iref{expr.new}, or
+\item
+when the explicit type conversion syntax\iref{expr.type.conv} is used.
```

Comment:
I'm not sure this last case is an "invoked implicitly" case. If we want to list this here, then perhaps we should also cover `new T()` and the various list-initialization cases?

Reply:
Well, I just bulletized the pre-existing text.

If you believe that "explicit type conversion" is already properly handled elsewhere, then we can strike it here.

Speaking of which, it seems the entire "implicitly invoked" paragraph talking about omitted initializers ought to be covered by [dcl.init] p11 and p7.  In particular, p11 talks about overload resolution, so we get access checking for free.

It feels the entire paragraph can become a note.

Reply:
Turned into a note and shortened substantially.